code/modules/clothing/glasses/glasses.dm

```diff
@@ -395,6 +395,11 @@
 	tint = 3				//to make them blind
 	prescription_upgradable = 0
 
+/obj/item/clothing/glasses/sunglasses/blindfold/fake
+    name = "tattered blindfold"
+    flash_protect = 0
+    tint = 0
```

Comment:
I think the description got caught in the crossfire of the variable removal here, you probably want to add that back in.
Also this has spaces rather than tabs for indentation, so it'd be good to change those over too:
```suggestion
/obj/item/clothing/glasses/sunglasses/blindfold/fake
	name = "tattered blindfold"
	desc = "A see-through blindfold perfect for cheating at games like pin the stunbaton on the clown."
	flash_protect = 0
	tint = 0
```